databricks/koala/structures.py

```diff
@@ -146,6 +146,9 @@ class _Frame(object):
     def max(self):
         return _reduce_spark(self, F.max)
 
+    def abs(self):
```

Comment:
You can add the decorator `@derived_from(pd.DataFrame)`. This will add the documentation of the pandas.DataFrame object corresponding to that method automatically.

Reply:
I'm going to copy the docstring instead of using the one in Pandas.
Cause the pandas one has a bunch of info about:
```
For ``complex`` inputs, ``1.2 + 1j``, the absolute value is` :math:`\sqrt{ a^2 + b^2 }`.
And also to some examples on `pd.Timedelta('1 days')` - which I dont think spark supports
```